packages/hdwallet-trezor/src/trezor.ts

```diff
@@ -150,10 +150,6 @@ export class TrezorHDWalletInfo implements core.HDWalletInfo, core.BTCWalletInfo
     return Btc.btcGetAccountPaths(msg);
   }
 
-  public btcIsSameAccount(msg: Array<core.BTCAccountPath>): boolean {
```

Comment:
? part of `BTCWalletInfo` interface. seems like we should still have this in some form or fashion